account/update_nex_account_rpc.proto

```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+
+package account;
+
+option go_package = "./;grpc_account";
+
+// TODO - What should this actually update?
+message UpdateNEXAccountRequest {}
```

Comment:
Well it should at least update permissions for the account

Reply:
Lowkey forgot about that tbh